LinqExample/LinqExample/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+
+namespace LinqExample
+{
+    internal static class Program
+    {
+        /*
+         * Вариант 6.
+         * 
+         * 1. Написать метод, который по заданному массиву целых чисел получает новый массив сумм цифр
+         * различных положительных чётных чисел исходного массива. Элементы нового массива должны идти в порядке
+         * убывания и тоже быть попарно различными. Метод должен быть написан с использованием только одного оператора.
+         * 
+         * 2. Написать метод, который для каждой пары «год–месяц», присутствующей в записях базыданных определяет
+         * количество клиентов, которые посещали центр в указанное время, а затем выводит эту информацию на консоль
+         * (вначале выводится год, затем месяц, затем количество клиентов). Сведения о каждой паре «год–месяц» выводить
+         * на новой строке и упорядочивать по убыванию номера года, а для одинакового номера года — по возрастанию
+         * номера месяца.
+         */
+        private static void Main()
+        {
+            var numbers = new[] {-1, 0, 1, 2, 3, 4, 5, 100, 101, 102, 103, 105};
+            Console.WriteLine(string.Join(", ", GetOrderedNumbers(numbers, 2)));
```

Comment:
Убрать второй аргумент в вызове метода, если второй параметр метода отсутствует.
```suggestion
            Console.WriteLine(string.Join(", ", GetOrderedNumbers(numbers)));
```